README.md

```diff
@@ -1,8 +1,71 @@
-# onix-docs
-Documentation, technical reference manual, and examples for the Bonsai ONI packages
+# Onix1 Bonsai Docs
 
-# Building docs locally
+Documentation for the Onix1 Bonsai packages.
 
-To build everything, including the Bonsai workflows, and view locally, run
+These docs are built on [docfx](https://dotnet.github.io/docfx/index.html).
+
+## Initialize the Documentation for Building Locally
+
+These actions only need to be performed once when setting up the repo locally.
+
+Download [dotnet](https://dotnet.microsoft.com/en-us/download).
+
+To clone the remote repository into the active directory, run:
+
+`git clone https://github.com/open-ephys/onix1-bonsai-docs`
+`cd onix1-bonsai-docs`
+
+To pulls in the latest files from the submodules according to the commit that the submodules point to, run:
+
+`git submodule update --recursive --init`
+
+To configure dotnet, run:
+
+`dotnet tool restore --configfile .\.bonsai\NuGet.config `
+
+In particular, restoring the config file configures docfx version and docfx companion tools such as [DocLinkChecker](https://github.com/Ellerbach/docfx-companion-tools/tree/main/src/DocLinkChecker).
+
+To make the `docfx` command available after restoring the config file from the previous step, run:
+
+`dotnet tool restore`
```

Comment:
This is the only line I'm not sure is needed, doesn't the previous `restore` command pull everything in already?

Reply:
When I try to run `dotnet docfx` after running the previous `restore` command for the first time, it prints this:

![Screenshot 2024-08-09 131211](https://github.com/user-attachments/assets/4deb2a76-73a7-4190-ad27-032a07f88797)

I think you're right though. I don't think I should have to run `dotnet tool restore` _again_. dotnet might not be restoring correctly on my system when I run the first `restore` command. Something's amiss.